Upgrade guide editorial review (#5691)
Particular/docs.particular.net: nservicebus/upgrades/7to8/message-contracts.md

 - 8
---

> NServiceBus version 8 does not support .NET Standard as a target framework. Therefore, message contract assemblies that target `netstandard2.0` and reference NServiceBus must be updated. This guide shows possible approaches. A general guidance is available in the [sharing message contracts](/nservicebus/messaging/sharing-contracts.md) documentation.

Reviewer: I'm not sure about "A general guidance". It doesn't read quite right.

> NServiceBus version 8 does not support .NET Standard as a target framework. Therefore, message contract assemblies that target `netstandard2.0` and reference NServiceBus must be updated. This guide shows possible approaches. A general guidance is available in the [sharing message contracts](/nservicebus/messaging/sharing-contracts.md) documentation.

Reviewer 2: ```suggestion
NServiceBus version 8 does not support .NET Standard as a target framework. Therefore, message contract assemblies that target `netstandard2.0` and reference NServiceBus must be updated. This guide shows possible approaches. General guidance is available in the [sharing message contracts documentation](/nservicebus/messaging/sharing-contracts.md).
```

let's rollback that change then.

Note: When using [unobtrusive mode](/nservicebus/messaging/unobtrusive-mode.md), the contracts assembly doesn't require a reference to NServiceBus and therefore is not affected. Unobtrusive message contracts can continue to target .NET Standard.

> ## Change to specific target Platform

Reviewer 2: ```suggestion
## Change to specific target Framework
```

target framework seems to be the better title then

> If all endpoints target the same target framework (e.g., .NET Core 3.1), the message contracts assembly can be changed accordingly to align all endpoint projects.

Reviewer: The section header uses the term platform so we should use that term in the text (or update the header).

`e.g.,` has a weird comma that I don't think should be there.

Target the same target framework feels redundant. Even fixing that, the sentence is kind of difficult to follow. I had to read it a few times to figure out what type of change it was recommending.

Maybe it's worth trying to reword it. Something like:

```suggestion
If all endpoints share the same target framework (e.g. .NET Core 3.1), the message contracts assembly can be changed from targeting .NET Standard to the same target framework as the endpoint projects.
```

Reviewer 2: > e.g., has a weird comma that I don't think should be there.

I was being reached just recently that the weird comma is supposed to be there ;)
